tests/src/python/providertestbase.py

```diff
@@ -240,6 +246,26 @@ def runGetFeatureTests(self, provider):
         values = [f['pk'] for f in self.vl.getFeatures(request)]
         self.assertEqual(values, [4])
 
+    def runPolyGetFeatureTests(self, provider):
+        assert len([f for f in provider.getFeatures()]) == 4
+
+        # geometry
+        self.assert_query(provider, 'xmin($geometry) < -70', [1, 3])
```

Comment:
Ah yeah - what I meant (and should have said) was that these tests for xmin/etc should also be added for the point layer so that it's assured that if other (non postgis) providers compile xmin/etc that the compilation works for point layers.